README.md

```diff
@@ -72,24 +71,26 @@ docker_image_manifest_path = ""   # Path to manifest in docker image
 The default configuration of _dbtwiz_ will be installed the first time you run it, but you
 may want to adjust some settings from the get-go to fit your environment.
 
+The config settings are stored in a file `config.toml` in the `dbtwiz` folder within
+your user's app settings directory. In a GitHub Codespace or in a local Linux environment
+the file is located at `~/.config/dbtwiz/config.toml`.
+
 #### Dark mode
 If you're using a dark background colour in your terminal, you should configure _dbtwiz_ to
 use bright colours for highlighting in previews and elsewhere to make the text more readable.
 
-Run the following command to switch from default light mode to dark mode:
-```shell
-$ dbtwiz config theme dark
-```
+Edit the config file to include `theme = dark` to achieve this. You will also need to
+delete the contents of the `.dbtwiz/models` directory within the project to rebuild the model
+info after changing theme.
 
 #### Preview formatter
 
 By default, _dbtwiz_ uses the command _fmt_ tool to format text in the preview window when
```

Comment:
I don't think we can/should use `fmt` (or `cat`) by default unless there is some check that verifies that they're supported.
On my windows machine, I get these types of error:
>'fmt' is not recognized as an internal or external command, operable program or batch file.
>'cat' is not recognized as an internal or external command, operable program or batch file.

Reply:
Should choose sensible default based on platform now.